twisted/conch/ssh/userauth.py

```diff
@@ -414,7 +414,7 @@ def ssh_USERAUTH_FAILURE(self, packet):
         removing methods that have already succeeded. We then call
         C{self.tryAuth} with the most preferred method.
 
-        @param packet: the L{MSG_USERAUTH_FAILURE} payload.
+        @param packet: the C{MSG_USERAUTH_FAILURE} payload.
```

Comment:
Please try to add documentation :) ... it should be a good exercise of reading RFC documentation :)